pandas/core/computation/ops.py

```diff
@@ -509,7 +509,7 @@ def _disallow_scalar_only_bool_ops(self) -> None:
 
 
 def isnumeric(dtype) -> bool:
-    return issubclass(np.dtype(dtype).type, np.number)
+    return getattr(dtype, '_is_numeric', False) or issubclass(np.dtype(dtype).type, np.number)
```

Comment:
```suggestion
        return getattr(dtype, "_is_numeric", False) or issubclass(
        np.dtype(dtype).type, np.number
    )
```
This is needed for `pre-commit` to success